.github/workflows/docs.yaml

```diff
@@ -41,9 +42,11 @@ jobs:
           echo "flink_branch=${currentBranch}"
           echo "flink_branch=${currentBranch}" >> ${GITHUB_ENV}
           if [ "${currentBranch} = "main" ]; then
-            echo "flink_alias=release-1.0" >> ${GITHUB_ENV}
-          elif [ "${currentBranch} = "release-0.1" ]; then
+            echo "flink_alias=release-1.1" >> ${GITHUB_ENV}
```

Comment:
It seems that the step `upload documentation alias` not really works.

https://github.com/apache/flink-kubernetes-operator/runs/6516829906?check_suite_focus=true

Reply:
strange, maybe @mbalassi has a good idea what's going on here